devscripts/run_tests.bat

```diff
@@ -1,4 +1,4 @@
 @echo off
 
 >&2 echo run_tests.bat is deprecated. Please use `devscripts/run_tests.py` instead
-py %~dp0run_tests.py %~1
+python %~dp0run_tests.py %~1
```

Comment:
It is better to use `py` on windows
```suggestion
py %~dp0run_tests.py %~1
```

https://docs.python.org/3/using/windows.html#launcher

Reply:
I guess this would have broken my workflow bc i didnt install it, however i dont care. Imo we could remove it just as well